### PR TITLE
 changed instance type to t2 medium

### PR DESCRIPTION
**Pull request by 582Bamfo**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ variable "name" {
  }
 
  variable "instance_type" {
-  default =  "t2.small"
```
Comment (582Bamfo) [inline variables.tf]: why default Dan?

Comment (582Bamfo) [reply variables.tf]: sorry I missed that . will fix it straight away

Comment (Thoughtbloom677) [reply variables.tf]: Looks great. Proceed

File: variables.tf [hunk 2]
```diff
@@ -10,7 +10,7 @@ variable "name" {
...
+  default =  "t2.medium"
   type = string
```
Comment (yeakvan5050) [inline variables.tf:14]: why did you use string

File: variables.tf (hunk 1 above)
Comment (AmandaDeveny) [reply variables.tf]: Thank you for the merge

**Review by yeakvan5050 — COMMENTED** [review]
it looks good so far

File: variables.tf (hunk 2 above)
Comment (582Bamfo) [reply variables.tf:14]: that is the data type

**Pull request merged**